packages/cli/src/commands/env/ls.ts

```diff
@@ -1,15 +1,11 @@
 import chalk from 'chalk';
 import ms from 'ms';
 import { Output } from '../../util/output';
-import {
-  ProjectEnvTarget,
-  Project,
-  ProjectEnvVariable,
-  ProjectEnvType,
-} from '../../types';
+import { Project, ProjectEnvVariable, ProjectEnvType } from '../../types';
 import Client from '../../util/client';
 import formatTable from '../../util/format-table';
-import getEnvVariables from '../../util/env/get-env-records';
+import getEnvRecords from '../../util/env/get-env-records';
+import formatEnvTarget from '../../util/env/format-env-target';
```

Comment:
Can we use aliases for the base folders?

Reply:
I don't think `ncc` supports that TS feature